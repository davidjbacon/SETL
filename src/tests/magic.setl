PRINT('Initial setting of MAGIC: ',MAGIC);
s := 'Dav*d';
p := 'av*';
PRINT('Subject string: ',PRETTY s);
PRINT('Pattern string: ',PRETTY p);
MAGIC := TRUE;
PRINT('With MAGIC TRUE, s(p) =',PRETTY (s(p) ? '<no match>'));
MAGIC := FALSE;
PRINT('With MAGIC FALSE, s(p) =',PRETTY (s(p) ? '<no match>'));
