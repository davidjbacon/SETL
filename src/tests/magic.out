Initial setting of MAGIC:  #T
Subject string:  'Dav*d'
Pattern string:  'av*'
With MAGIC TRUE, s(p) = 'av'
With MAGIC FALSE, s(p) = 'av*'
